### PR TITLE
Implemented status and pin assignments

### PR DESCRIPTION
**Pull request by jb1024**

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -15,7 +15,7 @@ framework = arduino
 
 lib_extra_dirs = ./common
 
-upload_port = /dev/cu.usbmodem14701
+;upload_port = /dev/cu.usbmodem14101
```
Comment (rvcaugit) [inline platformio.ini:19]: Can you add mine as comment, then I do not have to look it up every time ;)
upload_port = /dev/cu.usbmodemFD1411

File: src/PinAssignments.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <cstdint>
+
+#include <Pin.h>
+
+const struct SPins
+{
+  SPin Led0 = {13, false};
```
Comment (rvcaugit) [inline src/PinAssignments.h:9]: invert = false
Suggest to use enumerate: ACTIVE_HIGH and ACTIVE_LOW?
maybe rename invert to activeState?

**Review by rvcaugit — COMMENTED** [review]
Some minor comments/suggestions

File: platformio.ini (hunk 1 above)
Comment (jb1024) [reply platformio.ini:19]: - [ ] Add ralphs device

File: src/PinAssignments.h (hunk 2 above)
Comment (jb1024) [reply src/PinAssignments.h:9]: - [ ] Improve invert naming

**Pull request merged**